src/app/rpmostree-pkg-builtins.c

```diff
@@ -69,11 +85,26 @@ pkg_change (RPMOSTreeSysroot *sysroot_proxy,
                                 cancellable, &os_proxy, error))
     goto out;
 
+  if (local_pkgs != NULL && local_pkgs->len > 0)
+    {
+      handles = g_ptr_array_new ();
+      fdl = g_unix_fd_list_new ();
+      for (guint i = 0; i < local_pkgs->len; i++)
+        {
+          int fd = GPOINTER_TO_INT (local_pkgs->pdata[i]);
+          int idx = g_unix_fd_list_append (fdl, fd, error);
+          if (idx < 0)
+            goto out;
+          close (fd);
```

Comment:
At this point we also need to set `local_pkgs->pdata[i] = GINT_TO_POINTER(-1)` right?